integrations/appboy/lib/index.js

```diff
@@ -47,6 +47,10 @@ var Appboy = (module.exports = integration('Appboy')
   .tag(
     'v2.7',
     '<script src="https://js.appboycdn.com/web-sdk/2.7/appboy.min.js">'
+  )
+  .tag(
+    'v3.1',
```

Comment:
Added new 3.1 option to settings

![image](https://user-images.githubusercontent.com/36717/102635904-cd02f900-4108-11eb-9b68-a9794202d3a8.png)